src/components/header/Header.tsx

```diff
@@ -1,19 +1,48 @@
 'use client';
 
+import LocaleSwitcher from './LocaleSwitcher';
+import LogoKarlsruhe from '../../public/Logo_Digital_Karlsruhe-rechts_rgb.svg';
+import LogoKarlsruheSmall from '../../public/Logo_Digital_Karlsruhe-rechts_rgb_small.svg';
+import { usePathname } from 'next/navigation';
 import { useShowNavigation } from '../navigation/NavigationProvider';
+import { useTranslations } from 'next-intl';
 
 export default function Header() {
   const { show, setShow } = useShowNavigation();
+  const t = useTranslations('Header');
+  const showSearchbar = !usePathname().endsWith('/home');
   return (
-    <>
-      <button
-        className="btn btn-secondary d-block d-md-none"
-        onClick={(e) => {
-          setShow(!show);
-        }}
-      >
-        <i className="bi bi-list" />
-      </button>
-    </>
+    <div className="bg-white pb-3">
+      <div className="container-md">
+        <div className="d-flex flex-row d-md-none fs-5 m-2">
+          <button
+            className="d-block btn btn-secondary"
+            onClick={(e) => {
+              setShow(!show);
+            }}
+          >
+            <i className="bi bi-list" />
+          </button>
+          <div className="d-block flex-fill align-content-center p-2 ps-4">{t('title')}</div>
+          <div className="d-block justify-content-end" style={{ height: '50px' }}>
+            <LogoKarlsruheSmall />
```

Comment:
Please use the [Next.js Image Cmoponent](https://nextjs.org/docs/pages/api-reference/components/image). It offers built-in image optimization in addition to other advantages.

Also see https://blog.logrocket.com/import-svgs-next-js-apps/#load-svgs-next-js-using-next-image-component for usage with SVGs.

Reply:
Ah, it wasn't working for me before, because I wasn't using static imports 😅